src/components/SharedComponents/ObservationLocation.js

```diff
@@ -14,25 +15,18 @@ type Props = {
 const ObservationLocation = ( { observation, margin }: Props ): React.Node => {
   const { t } = useTranslation();
 
-  let locationName = observation?.place_guess;
-
-  if (
-    !locationName
-    // Check for undefined or null, Not 0
-    && observation?.latitude != null
-    && observation?.longitude != null
-  ) {
-    locationName = `${observation.latitude}, ${observation.longitude}`;
-  } else if ( !locationName ) {
-    locationName = t( "Missing-Location" );
+  let displayLocation = checkCamelAndSnakeCase( observation, "placeGuess" );
+  if ( !displayLocation && observation.latitude ) {
```

Comment:
I think this was brought up in another comment, but this if statement wouldn't run on any location on the equator since `0` would be falsey.  `foo != null` would be `false` for `null` or `undefined` but would allow `0` to pass

Reply:
Made a test case for the above.